guide/src/reference/static-js-objects.md

```diff
@@ -65,6 +70,20 @@ extern "C" {
 }
 ```
 
+## Undeclared
+
+When accessing an in JS undeclared value, it will throw in JS. This can be accounted for by using `Option`.
+
+```rust
+extern "C" {
+    type Crypto;
+    #[wasm_bindgen(thread_local_v2, js_name = crypto)]
+    static CRYPTO: Option<Crypto>;
+}
+```
+
+If `crypto` is undeclared in JS, it will simply return `None` in Rust.
```

Comment:
I wrote a bit more about the feature. The main things I added are to mention that `null` and `undefined` also map to `None` and that namespaces work as well.

I also gave this feature a name: optional statics. This will hopefully make it easier to teach the feature.

Unfortunately, I can't use GH suggestions to suggest the change, because those suggestions use MD syntax... so here it is:

    ## Optional statics

    Sometimes, the JavaScript value you're trying to access may not be available, e.g. because the browser does not support it. In this case, you can use `Option<T>` to handle this. Example:

    ```rust
    #[wasm_bindgen]
    extern "C" {
        type Crypto;
        #[wasm_bindgen(thread_local_v2, js_name = crypto)]
        static CRYPTO: Option<Crypto>;
    }
    ```

    If `crypto` is not defined in JavaScript, it will simply return `None` in Rust.

    Note that `None` will also be returned if the value of the `crypto` variable is nullish (`null` or `undefined`).

    Namespaces are handled similarly. Example:

    ```rust
    #[wasm_bindgen]
    extern "C" {
        #[wasm_bindgen(thread_local_v2, js_name = port, js_namespace = server)]
        static PORT: Option<u32>;
    }
    ```

    `PORT` will only return a `Some` value in Rust if `server.port` is set to a number in JavaScript. If `server` is not defined or has a nullish value, it will return `None`. If the `port` property is not defined on the `server` object or is set to a nullish value, it will also return `None`.